runtime/compiler/control/J9Options.cpp

```diff
@@ -1069,8 +1069,8 @@ bool J9::Options::showPID()
 #if defined(J9VM_OPT_JITSERVER)
 static std::string readFileToString(char *fileName)
    {
-   I_32 fileId = j9jit_fopen_existing(fileName);
-   if (fileId == -1)
+   TR::FILE *f = j9jit_fopen(fileName, "r", false, false);
+   if (!f)
       return "";
    const uint32_t BUFFER_SIZE = 4096; // 4KB
    const uint32_t MAX_FILE_SIZE_IN_PAGES = 16; // 64KB
```

Comment:
Not really related to this PR, but can we guarantee that certificate files are never larger than 64 KB?

Reply:
At the time, 64KB seemed like a reasonable limit.
The limit avoids a scenario where somebody passes in a very large (unrelated) file, either by mistake or intentionally, and cause the JVM to allocate huge amounts of memory in the string we build.
Having said that, I think the caller should abort the JVM with a message if the user specifies a certificate/key but we cannot read it.

Reply:
Hm, so aborting JVM from `JITServerParseCommonOptions` with an error message seems to require the following:
- Make `JITServerParseCommonOptions` return a bool, indicating whether JITServer options were parsed correctly. If false is returned, return false from `fePreProcess` which will abort JVM.
- Use `j9nls_printf` to print an error message. @mpirvu, does this mean I would have to add a new NLS error, or should I just use a regular `printf`

Reply:
It may be better to use j9nls because it is translated in many languages.
Make it generic, so that it can be used in other places as well. This way we can say this is not necessarily JITServer related. 
Something like: File not found or too large